libs/barista-components/chart/src/heatfield/chart-heatfield.ts

```diff
@@ -223,6 +223,14 @@ export class DtChartHeatfield extends _DtHeatfieldMixinBase
     this.active = !this.active;
   }
 
+  /**
+   * Closes the context dialog
+   * @internal
+   */
+  close(): void {
+    this.active = false;
+  }
+
```

Comment:
This method is doing the same thing as setting active to false on the instance. I'd rather keep the API surface small and not add this method to do the same thing

Reply:
Alright, valid point

Reply:
done